include/device.h

```diff
@@ -208,7 +208,7 @@ struct device {
 	void * const data;
 #ifdef CONFIG_DEVICE_POWER_MANAGEMENT
 	/** Power Management function */
-	int (*device_pm_control)(struct device *device, uint32_t command,
+	int (*device_pm_control)(struct device *dev_def, uint32_t command,
```

Comment:
Can you please share the rationale for appending with _def? What does it stand for?

Reply:
fwiw - I remember seeing `struct device *dev` used frequently in gpio drivers, example: https://github.com/zephyrproject-rtos/zephyr/blob/master/drivers/gpio/gpio_intel_apl.c

Reply:
just call it 'dev'

Reply:
+1 I think I have suggested the same thing before as well.

Reply:
changed it to dev